### ISSUE TITLE
Replace various slider/carousel libraries with flickity.js

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1224]
## Context

Rather than use the three different libraries we currently use for carousel type content (sliders, article sliders, simple horizontal sequence) we can use a single library, Flickity.js

**Comment by cathysnider** [comment]
Acceptance Criteria:
- [ ] a slider that has been created still works in all slider styles
- [ ] an article slider that has been created still works. 
- [ ] The previous and next buttons look good.
- [ ] Sliders display well in all themes.
- [ ] Can create content in the new code

https://www-dev.colorado.edu/p1bfbe9ace81/ (default theme: Spirit. Looks good)

https://www-dev.colorado.edu/p19b149c4244/ (attempt to create slider block causes site to white-screen; enabling News bundle throws error; see screenshot)

https://www-dev.colorado.edu/p11b135d6b1b/ Article Slider buttons look bad. Clearing cache doesn't seem to solve problem. See screenshot. (NOTE: CHECK THIS SITE AGAIN IN A FEW HOURS)
![screen shot 2017-11-29 at 1 42 27 pm](https://user-images.githubusercontent.com/638275/33398198-61935406-d50b-11e7-995e-1d4d86ae215c.png) 

note that admin/appearance is back
![flickityappearance](https://user-images.githubusercontent.com/638275/33396889-3188c8b2-d507-11e7-8cb9-900f486ff5b7.jpg)

**Comment by kevincrafts** [comment]
Looks like the update hook to enable the cu_article_slider module didn't actually enable the slider - I manually enabled it - this may have been caused by the site being installed with a more recent version of express, then switched to a previous branch.

Anyways, the article slider block appears as it should:

https://www-dev.colorado.edu/p11b135d6b1b/block/article-slider
![screen shot 2017-11-30 at 1 52 01 pm](https://user-images.githubusercontent.com/349713/33454301-ae58bacc-d5d5-11e7-98e8-8383f62190a9.jpg)

**Comment by cathysnider** [comment]
Dev merged into Feature Branch 1224; old code and sites removed, new code spun up and new site created. The wonkiness continues:
- Article Slider messed up: https://www-dev.colorado.edu/p1ae8b5e07c6/news
- Content Overlay slider DOES have better contrast, so that's good
- The new slider Thumbnail styles are not an option any longer.

![screen shot 2017-12-01 at 11 13 24 am](https://user-images.githubusercontent.com/638275/33497178-45ee165c-d68a-11e7-9767-fa755e94d345.png)

![screen shot 2017-12-01 at 11 25 22 am](https://user-images.githubusercontent.com/638275/33497207-605aa8ac-d68a-11e7-9e83-6909fd057c75.png)

**Comment by cathysnider** [comment]
Another one: https://www-dev.colorado.edu/p135ac7a2b04/

**Comment by cathysnider** [comment]
Another one: https://www-dev.colorado.edu/p1b456470874/ using latest commit of 5b9549b
The article Slider is not messed up. 
The regular slider is missing the new slider styles of Thumbnail, Thumbnail Overlay and Hero.

So we aren't quite there yet.

**Comment by kevincrafts** [comment]
Latest commits remove additional slider styles that appeared in a previous push.  Would like to hold those for a larger feature release.

**Comment by cathysnider** [comment]
One problem:
The article slider is not filtering: it is displaying all articles though it's being told to display only those one tag.  
https://www-dev.colorado.edu/p1691f67fda1/block/article-slider-post-update

**Comment by kevincrafts** [comment]
Not sure what's happening with that site but I can't really troubleshoot while various parts are white screening - https://www-dev.colorado.edu/p1691f67fda1/admin/config

**Comment by cathysnider** [comment]
When attempting to run the update script on a new Flickity Install: 
https://www-dev.colorado.edu/p1d905055864
Get this error: 

![screen shot 2018-02-14 at 1 31 18 pm](https://user-images.githubusercontent.com/638275/36227042-00f1d5aa-118d-11e8-8261-ebf9c446e301.png)

and Configuration page white screens. 
https://www-dev.colorado.edu/p1d905055864/admin/config

**Comment by cathysnider** [comment]
@kevincrafts would you pls merge dev into this branch.

**Comment by cathysnider** [comment]
Acceptance Criteria:

 - [ ] a slider that has been created still works in all slider styles
 - [ ] an article slider that has been created still works.
 - [ ] The previous and next buttons look good. FAILING ON ONE SITE; RE-CHECK LATER
 - [ ] Sliders display well in all themes.
 - [ ] Can create content in the new code
 - [ ] A regular slider and an article slider can exist on the same page.
